app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'benefits_intake_service/service'
+
+class BenefitsIntakeStatusJob
+  include Sidekiq::Job
+
+  def perform
+    pending_form_submission_ids = FormSubmission
+                                  .joins(:form_submission_attempts)
+                                  .where('form_submission_attempts.aasm_state = ?', 'pending')
+                                  .map(&:benefits_intake_uuid)
+    response = BenefitsIntakeService::Service.new.get_bulk_status_of_uploads(pending_form_submission_ids)
+    handle_response(response)
+  end
+
+  private
+
+  def handle_response(response)
+    response.body['data'].each do |submission|
+      if submission.dig('attributes', 'status') == 'error' || submission.dig('attributes', 'status') == 'expired'
+        handle_failure(submission)
+      elsif submission.dig('attributes', 'status') == 'vbms'
+        handle_success(submission)
+      end
+    end
+  end
+
+  def handle_failure(submission)
+    form_submission = FormSubmission.find_by(benefits_intake_uuid: submission['id'])
+    form_submission_attempt = form_submission
+                              .form_submission_attempts
+                              .where(aasm_state: 'pending')
+                              .order(created_at: :asc)
+                              .last
```

Comment:
This grabs the last pending form submission attempt. Theoretically we shouldn't have multiple form submission attempts that are pending since it should fail before we make another attempt. But if we want to do more defensive coding around this, I'm all ears!